app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -305,5 +323,43 @@ private boolean noDuplicateNumber() {
         }
         return noDuplicate;
     }
+
+    /**
+     * Set focus to the next EditText after picking contact
+     */
+    public void setFocusOnNextView() {
```

Comment:
Tags for buttons were already added by the developer who added the contact retrieval feature. I am not trying to find the index of the selected button. I am trying to find the index of the next view which doesn't have any numbers registered.